Documentation/LanguageMenu/Index.rst

```diff
@@ -17,18 +17,14 @@ depending on translation availability, i.e.:
 - Missing translation
 - Missing translation - currently selected
 
-This kind of menu can be made by TypoScript and the
-:ref:`HMENU content object <t3tsref:cobj-hmenu>`
+This kind of menu can be made by either using a
+:ref:`LanguageMenuProcessor on a FLUIDTEMPLATE <t3tsref:cobj-fluidtemplate>`
+or using the :ref:`HMENU content object <t3tsref:cobj-hmenu>`
 with special type :ref:`language <t3tsref:hmenu-special-language>`.
 
-Such a menu answers to the above requirements by using four
-different item states, respectively "NO", "ACT", "USERDEF1"
-and "USERDEF2".
-
-
 .. _language-menu-tmenu:
 
-TMENU example
+LanguageMenuProcessor example
 =============
```

Comment:
Underline should be same length as title. Otherwise may result in rendering problem or warnings, see https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-us/WritingReST/CommonPitfalls/Headers.html